### DISCUSSION TITLE
Passing Audio Data Input to Deepgram Voice Agent WebSocket

### DISCUSSION THREAD

**Discussion by Divyeshranagithub** [deepgram/community#1024] [General help]
Hi,
In my FastAPI project, I am using the Deepgram Voice Agent WebSocket:
`wss://agent.deepgram.com/agent`

However, I am unable to find documentation on how to pass audio data as input to this WebSocket.

For comparison, when working with OpenAI's WebSocket `wss://api.openai.com/v1/realtime?model=gpt-4o-realtime-preview-2024-10-01`, we use the following code snippet to send audio data:
```python
audio_append = {
    "type": "input_audio_buffer.append",
    "audio": data['media']['payload']
}
await openai_ws.send(json.dumps(audio_append))
``` 
Could you please guide me on how to format and send audio data to the Deepgram Voice Agent WebSocket? Any examples or references to relevant documentation would be greatly appreciated.

Thank you!

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Reply by Divyeshranagithub** [reply]
i am not able to do connect github with deepgram

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- A request ID that triggered your error or issue.

**Reply by Divyeshranagithub** [reply]
where is it
